stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/chat/info/ChatInfoFragment.kt

```diff
@@ -0,0 +1,90 @@
+package io.getstream.chat.ui.sample.feature.chat.info
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.viewModels
+import androidx.navigation.fragment.findNavController
+import androidx.navigation.fragment.navArgs
+import io.getstream.chat.ui.sample.R
+import io.getstream.chat.ui.sample.common.getFragmentManager
+import io.getstream.chat.ui.sample.common.initToolbar
+import io.getstream.chat.ui.sample.common.navigateSafely
+import io.getstream.chat.ui.sample.databinding.FragmentChatInfoBinding
+import io.getstream.chat.ui.sample.feature.chat.ChatViewModelFactory
+
+class ChatInfoFragment : Fragment() {
+
+    private val args: ChatInfoFragmentArgs by navArgs()
+    private val factory: ChatViewModelFactory by lazy { ChatViewModelFactory(args.cid) }
+    private val viewModel: ChatInfoViewModel by viewModels { factory }
+    private val adapter: ChatInfoAdapter = ChatInfoAdapter()
+
+    private var _binding: FragmentChatInfoBinding? = null
+    private val binding get() = _binding!!
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        _binding = FragmentChatInfoBinding.inflate(inflater, container, false)
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        initToolbar(binding.toolbar)
+        initAdapter()
+        viewModel.bindView(this, viewLifecycleOwner)
+    }
+
+    override fun onDestroyView() {
+        super.onDestroyView()
+        _binding = null
```

Comment:
Canonnically, it's better to run user's code before invoking `super.onDestroyView()`, for example
```kotlin
fun onStart() {
  super.onStart()
  // your code here
}

fun onStop() {
 // your code here
 super.onStop()
}

Reply:
We do it this way everywhere, it's from Google's sample code 😅 

https://developer.android.com/topic/libraries/view-binding#fragments

Reply:
For this specific task of setting something to `null`, it doesn't really matter where we do it afaik, but it's true that otherwise it's logically better to do it before the `super` call